jobs/mprsyncup

```diff
@@ -147,6 +153,17 @@ else
     ${GIT} clone -q --depth 1 "${PORTSURL}" "${PORTS}"
 fi
 
+#
+# Update release/contrib
+#
+
+if [ -d "${CONTRIB}"/.git ]; then
+    cd "${CONTRIB}"
+    ${GIT} -C "${CONTRIB}" pull -q
```

Comment:
I guess `-C "${CONTRIB}"` isn't necessary, since we already `cd "${CONTRIB}"`.

Reply:
Thank you. I have addressed this.